Tools/CISupport/ews-build/steps.py

```diff
@@ -91,13 +91,15 @@ def pr_url(cls, pr_number, repository_url=None):
         return '{}/pull/{}'.format(repository_url, pr_number)
 
     @classmethod
-    def commit_url(cls, sha, repository_url=None):
+    def commit_url(cls, sha, repository_url=None, pr_number=None):
```

Comment:
Might be a good idea to add unit-test for this

Reply:
Added tests.